api-metastore/src/main/java/org/zalando/nakadi/controller/util/CursorUtil.java

```diff
@@ -0,0 +1,55 @@
+package org.zalando.nakadi.controller.util;
+
+import org.zalando.nakadi.domain.NakadiCursor;
+import org.zalando.nakadi.domain.NakadiCursorLag;
+import org.zalando.nakadi.exceptions.runtime.CursorConversionException;
+import org.zalando.nakadi.exceptions.runtime.InternalNakadiException;
+import org.zalando.nakadi.exceptions.runtime.InvalidCursorException;
+import org.zalando.nakadi.service.CursorConverter;
+import org.zalando.nakadi.service.CursorOperationsService;
+import org.zalando.nakadi.view.Cursor;
+import org.zalando.nakadi.view.CursorLag;
+
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class CursorUtil {
+
+    public static Stream<CursorLag> toCursorLagStream(final List<Cursor> cursorList, final String eventTypeName,
```

Comment:
I think this method can be added to `CursorOperationsService` as a non-static one.

Reply:
Also adding a `CursorConverter` instance to `CursorOperationsService` will simplify the code.

Reply:
I refactored `CursorOperationsService` to include these methods but I have still kept `CursorConverter` as method argument because its only needed for this one method.